compass/compass.js

```diff
@@ -1,7 +1,10 @@
-﻿API.onUpdate.connect(function (sender) {
+var x;
+var y;
 
-    var x = API.getGameplayCamDir().X;
-    var y = API.getGameplayCamDir().Y;
+API.onUpdate.connect(function (sender) {
+
+    x = API.getGameplayCamDir().X;

```

Comment:
even if i do that, i would still need to call X and Y of the variable that i will assign getGameplayCamDir to